### ISSUE TITLE
idea: search only tags

### ISSUE THREAD

**Issue by edmundlaugasson** [aardappel/treesheets#69]
Trying to search only tags but F6 it cycles not only tags but all cells with same content. In that sense handling large information would be quite challenging when trying to search with F3. It finds all words as search string but to narrow down the search scope it would be useful if there could be an option to search only tags.

**Comment by aardappel** [comment]
Wouldn't the solution be to use more unique tags? That way F3 (and F6) become more useful.

**Comment by edmundlaugasson** [comment]
Currently regardless of tags F3 will search all words across the current document. F6 needs entering first same word somewhere into sheet (which is not comfortable) in order to search. Another issue would be that I cannot add tag to cell other than the cell content is - there is only a choice to set cell content as tag. Probably this is good idea and passes better with whole program concept. Just noticed from other environments that regardless of entered information there is separate tag field where can be defined one or more tags. Then if I search by tag and will find lots of results, would be useful to filter the results in turn. This is the challenging task to find information from large amount of data. Once I was enthused about a program called "The Brain" as it had a Linux version (in 8th version) which they do not have anymore in [next (9th) version](http://www.thebrain.com/products/thebrain/thebrain9/) and are not planning as well. The idea how they manage large amount of information is just a search field where I start typing information and it starts search as I type. Somehow they can manage large amount of data in that way, that it is not slow.

**Comment by aardappel** [comment]
F6 is for hopping between the same tags or titles already in the sheet, so it shouldn't be used for searching things you have to type in.

I agree search could have more options.. I tried to keep the UI as simple as possible.

**Comment by edmundlaugasson** [comment]
I agree - UI simplicity is good, very good. What about adding tag search? If not make yet another search field then e.g. option to switch search field between "search" (e.g. default) or "search tag". Perhaps it would then possible to use same F3 shortcut key which applies to search results according previous selection - "search" or "search tag".
